.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,3 +1,3 @@
 ## Contributor Checklist:
 
-* [ ] I have created a file in the `doc/newsfragments` directory (and read the `README.md` in that directory)
+* [ ] I have created a file in the `doc/newsfragments` directory *IF* it is a user-visible change (and make sure to read the `README.md` in that directory) 
```

Comment:
LGTM, but let's keep the line length at some reasonable limit like 100 characters. Long lines make files harder to edit as line wrapping is evil and people usually have several columns open side-by-side.

Also strip the trailing space.

Reply:
Fixed in 0f68c291 - I was using nano, not configured correctly.